salt/utils/cloud.py

```diff
@@ -2942,9 +2937,10 @@ def _salt_cloud_force_ascii(exc):
     if not isinstance(exc, (UnicodeEncodeError, UnicodeTranslateError)):
         raise TypeError('Can\'t handle {0}'.format(exc))
 
+    # Convert non-breaking space to space and convert en dash to dash
```

Comment:
I think I would prefer on each of these lines to know which character is what, rather than just summing it up.